File: oauth2-server/src/main/java/a/gleb/keycloaktojwt/db/repository/KeycloakAuthorizationSessionRepository.java
package a.gleb.keycloaktojwt.db.repository;

import a.gleb.keycloaktojwt.db.entity.KeycloakAuthorizationSession;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.lang.NonNull;

import java.util.List;
import java.util.UUID;

public interface KeycloakAuthorizationSessionRepository extends JpaRepository<KeycloakAuthorizationSession, UUID> {

    List<KeycloakAuthorizationSession> findAllByUsername(@NonNull String username);
}

File: oauth2-server/src/main/kotlin/a/gleb/oauth2server/db/repository/KeycloakAuthorizationSessionRepository.kt
package a.gleb.oauth2server.db.repository

import a.gleb.oauth2server.db.entity.KeycloakAuthorizationSession
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface KeycloakAuthorizationSessionRepository: JpaRepository<KeycloakAuthorizationSession, UUID>